projects/packages/identity-crisis/src/class-identity-crisis.php

```diff
@@ -190,6 +192,32 @@ public function wordpress_init() {
 		}
 	}
 
+	/**
+	 * Add the idc query arguments to the url.
+	 *
+	 * @param string $url The remote request url.
+	 */
+	public function add_idc_query_args_to_url( $url ) {
+		if ( ! is_string( $url ) ) {
```

Comment:
I think we should only add the idc query args if the site isn't already in an IDC. So, we should check the idc error option, or possibly validate the error option, before adding the query args. @zinigor - do you mind if I make that change in this PR? Or would you prefer a separate PR for that?